operations/mimir-mixin/config.libsonnet

```diff
@@ -212,11 +212,23 @@
     },
 
     // Whether autoscaling panels and alerts should be enabled for specific Mimir services.
-    autoscaling: {
-      querier_enabled: false,
-      querier_hpa_name: 'keda-hpa-querier',
-      ruler_querier_hpa_name: 'keda-hpa-ruler-querier',
-    },
+    autoscaling: [
+      {
+        name: 'querier',
```

Comment:
1. Moving away so that I can loop over the components.
2. Not sure what you mean by configuring it twice? But yes, your suggested structure would work well. It's really just moving the name field to a key, but it's much neater so will try to implement that.